plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -543,55 +561,110 @@ export class PersonState {
         // This is low-probability so likely won't occur on second retry of this block.
         // In the rare case of the person changing VERY often however, it may happen even a few times,
         // in which case we'll bail and rethrow the error.
-        await this.db.postgresTransaction('mergePeople', async (client) => {
-            try {
-                const [person, updatePersonMessages] = await this.db.updatePersonDeprecated(
-                    mergeInto,
-                    {
-                        created_at: firstSeen,
-                        properties: excludeProperties
-                            ? mergeInto.properties
-                            : this.updatedPersonProperties(mergeInto.properties),
-                        is_identified: true,
-                    },
-                    client
-                )
+        try {
+            let mergedPerson
+                // Keep the oldest created_at (i.e. the first time we've seen this person)
+            ;[kafkaMessages, mergedPerson] = await this.handleMergeTransaction(
```

Comment:
Makes sense. I gather previously you could have:

 1. enter postgresTransaction
 2. BEGIN
 3. writes to postgres
 4. sends messages to Kafka
 5. COMMIT raises integrity error

And now it will be:


 1. enter postgresTransaction
 2. BEGIN
 3. writes to postgres
 4. COMMIT raises integrity error

Sounds good